cookbook-react/src/recipes/inputs/santiago/Selector/styles.module.scss

```diff
@@ -0,0 +1,72 @@
+$color-gray: #E1E1E1;
+$color-black: #000;
+$color-white: #FFF;
+$color-blue: #003DD5;
```

Comment:
I think we add meaning to this const instead of color names. Like, `$active-color`, `$disabled-color`